package.json

```diff
@@ -20,7 +20,7 @@
     "build": "react-scripts build",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "lint": "eslint \"**/*.js\" --fix"
+    "lint": "eslint src --fix"
```

Comment:
How would this work in case the tests are moved away from the `src` components? Being actual functions and code, they should be linted as well as they also need to be maintained.

Reply:
Just change it to `eslint src test --fix`

Reply:
I didn't change the command yet because it would fail on an empty directory.

Check the commit message of 00d0a7adf7ab9ff28e6a401bec44bf3b3296b209